packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dedicatedCloud.service.js

```diff
@@ -339,14 +339,14 @@ class DedicatedCloudService {
   }
 
   getDatacenterInfoVm(serviceName, datacenterId, params = {}) {
-    return this.icebergQuery(
+    return this.icerbergUtils.icebergQuery(
```

Comment:
The error is not here but on the initial name `icerbergUtils`

Sorry It's me I think the origin of this problem